support-frontend/assets/components/headers/header/header.scss

```diff
@@ -213,3 +219,103 @@ $header-links-min-width: gu-span(12);
     }
   }
 }
+
+.component-header--display-checkout {
+  .component-header__wrapper {
+    max-width: 50rem;
+
+    @include mq($from: tablet, $until: desktop) {
+      max-width: 100%;
+      padding: 0;
+      margin: 0;
+    }
+
+    @include mq($from: leftCol) {
+      max-width: 60rem;
+    }
+
+    .component-header-topnav {
+      @include mq($until: tablet) {
+        justify-content: space-between;
+      }
+
+      @include mq($from: tablet, $until: desktop) {
+        justify-content: flex-end;
+      }
+
+      .component-header-topnav__utility {
+        display: none;
+      }
+
+      .component-header-topnav__checkout {
+        display: flex;
+        flex: 0 0 490px;
+        color: gu-colour(neutral-100);
+        @include gu-fontset-body-sans;
+        border-right: 1px solid gu-colour(brand-pastel);
+  
+        @include mq($until: tablet) {
+          flex: none;
+          border-right: none;
+        }
+      
+        .component-header-topnav--checkout-text {
+          border-top: 1px solid gu-colour(brand-pastel);
+          border-left: 1px solid gu-colour(brand-pastel);
+          padding: $gu-v-spacing;
+          width: 100%;
+          align-self: flex-end;
+
+          :nth-child(2) {
+            display: inline-block;
+            vertical-align: top;
+          }
+  
+          @include mq($until: tablet) {
+            border-left: none;
+            border-top: none;
+            margin-top: -16px;
+            padding-left: 2px;
```

Comment:
these two numbers look a bit magic, what are they doing? can we achieve this positioning in a cleaner way?

Reply:
You're right! The margin-top wasn't doing anything and the other one I have adjusted to be gu